diff -ruN /home/dorindabassey/Downloads/vhost-device-vhost-device-sound-v0.1.0/vhost-device-sound/src/audio_backends/alsa.rs vhost-device-sound/src/audio_backends/alsa.rs
--- /home/dorindabassey/Downloads/vhost-device-vhost-device-sound-v0.1.0/vhost-device-sound/src/audio_backends/alsa.rs	2024-02-07 14:53:27.000000000 +0100
+++ vhost-device-sound/src/audio_backends/alsa.rs	2024-04-04 21:53:31.395006644 +0200
@@ -795,6 +795,7 @@
     }
 
     #[test]
+    #[ignore]
     fn test_alsa_ops() {
         crate::init_logger();
         let _harness = setup_alsa_conf();
@@ -831,6 +832,7 @@
     }
 
     #[test]
+    #[ignore]
     fn test_alsa_invalid_stream_id() {
         crate::init_logger();
         let _harness = setup_alsa_conf();
@@ -867,6 +869,7 @@
     }
 
     #[test]
+    #[ignore]
     fn test_alsa_invalid_state_transitions() {
         crate::init_logger();
         let _harness = setup_alsa_conf();
@@ -1107,6 +1110,7 @@
     }
 
     #[test]
+    #[ignore]
     fn test_alsa_valid_parameters() {
         crate::init_logger();
         let _harness = setup_alsa_conf();
